ext/date/php_date.c

```diff
@@ -982,14 +982,14 @@ PHPAPI void php_date_set_tzdb(timelib_tzdb *tzdb)
 /* }}} */
 
 /* {{{ php_parse_date: Backwards compatibility function */
-PHPAPI zend_long php_parse_date(char *string, zend_long *now)
+PHPAPI zend_long php_parse_date(const char *string, zend_long *now)
 {
 	timelib_time *parsed_time;
 	timelib_error_container *error = NULL;
 	int           error2;
 	zend_long   retval;
 
-	parsed_time = timelib_strtotime(string, strlen(string), &error, DATE_TIMEZONEDB, php_date_parse_tzfile_wrapper);
+	parsed_time = timelib_strtotime((char *) string, strlen(string), &error, DATE_TIMEZONEDB, php_date_parse_tzfile_wrapper);
```

Comment:
Could you please also submit a PR to https://github.com/derickr/timelib, so we can drop these const casts in the future?

Reply:
I did not change it because I was worried whether this modification will affect other projects
but I will try to submit a PR soon

Reply:
Added `TODO` comment

Reply:
@nikic @derickr done: https://github.com/derickr/timelib/pull/82